components/nodemanager-service/pgdb/project_delete.go

```diff
@@ -0,0 +1,89 @@
+package pgdb
+
+import (
+	"context"
+
+	project_update_lib "github.com/chef/automate/lib/authz"
+	uuid "github.com/chef/automate/lib/uuid4"
+	log "github.com/sirupsen/logrus"
+)
+
+const deleteProjectNodeAssociation = `
+DELETE FROM nodes_projects np
+USING projects p
+WHERE p.id = np.project_id AND p.project_id = $1;
+`
+
+type ProjectDelete struct {
+	jobStatus      project_update_lib.JobStatus
+	jobStatusError error
+	running        bool
+	ID             string
+}
+
+func (db *DB) DeleteProjectTag(ctx context.Context,
+	projectTagToBeDeleted string) ([]string, error) {
+	jobUUID := uuid.Must(uuid.NewV4()).String()
+	log.Debugf("Running node manager DeleteProjectTag %v ID: %s", projectTagToBeDeleted, jobUUID)
+
+	db.ProjectDelete.start(jobUUID)
+
+	go db.deleteProject(projectTagToBeDeleted)
```

Comment:
We are just using goroutines here? What if the service crashes?

Reply:
If the service crashes then when we ping the service to check the `getJobStatus` we will see that the job failed. 

Looking at the `getJobStatus` https://github.com/chef/automate/pull/2424/files#diff-87e11a1c44a032343e1ce6928c55b7c7R65 I do need to change it some.

Reply:
On second thought I see another problem. This is only going to allow one project delete at a time. 🤔